docs/_data/toc.yml

```diff
@@ -28,6 +28,9 @@
   - title: "Sampler Visualization"
     url: "docs/using-turing/sampler-viz"
 
+  - title: "Turing Compiler Design"
```

Comment:
The compiler docs are fairly advanced and may not be useful for users. Maybe add an `Internal` or `For Developers` section, instead of putting compiler docs under `Using Turing`?

Reply:
I added a separate section to the TOC, so this will hold the interface guide and the compiler guide.